css/font-awesome.css

```diff
@@ -0,0 +1,2337 @@
+/*!
```

Comment:
I was mentioning this file

Reply:
That is for symbols. (FontAwesome delivers logos and symbols as fonts) FontAweosme which was earlier linked was outdated, so I linked the newer one. Nothing special changed other than the Google+ logo.